play-app/src/Components/ProdCard/ProdCard.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import "./ProdCard.css";
+import {
+  FiHeart,
+  FiStar
+} from "react-icons/fi";
+
+const ProdCard = ({prod}) => {
+   const{image,title,rating,newPrice,previousPrice,discount} = prod;
+  return ( <div >
+    <div className = "cards_withBadge" >
+    < div className = "header" >
+      < img className = "card-image_one" src = { image } alt = "" />
```

Comment:
Alt should have a value instead of empty string.

Reply:
will change